sbt

```diff
@@ -1,664 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I think it's just convenient for people that don't have `sbt` installed so that they can do `./sbt compile`. But I'm all up for removing it, I'm struggling to imagine people that would want to compile this project not having sbt installed